python-project-template/README.md.jinja

```diff
@@ -1,5 +1,10 @@
 # {{project_name}}
 
+[![Template](https://img.shields.io/badge/Template-LINCC%20Frameworks%20Python%20Project%20Template-brightgreen)](https://lincc-ppt.readthedocs.io/en/latest/)
+
 This project was automatically generated using the LINCC-Frameworks [python-project-template](https://github.com/lincc-frameworks/python-project-template).
 
+A repository badge was added to show that this project uses the python-project-template, however it's up to
+you whether or not you'd like to display it!
+
 For more information about the project template see the [readme](https://github.com/lincc-frameworks/python-project-template#readme) documentation.
```

Comment:
Maybe we should update this last link to point to readthedocs as well.

Reply:
Updated the last sentence to be the rtd link instead of the readme, if that's what you meant?

Reply:
yep, that's it!